src/main/services/settings.service.ts

```diff
@@ -19,7 +19,7 @@ const settingsStore = new FileStore<typeof settingsSchema>(path.join(CONFIG_PATH
     battlesHideLocked: false,
     battlesHideEmpty: true,
     logUploadUrl: "https://log.beyondallreason.dev/",
-    lobbyServer: "",
+    lobbyServer: "server4.beyondallreason.info",
     customServerList: [],
 });
```

Comment:
I'm confused why this default object here is needed at all when the defaults are already specified in the json schema in src/main/json/model/settings.ts.

This shouldn't be needed.